### PR TITLE
map some extra functions on the PamHandle

### PR DESCRIPTION
**Pull request by hardening**
Map pam_set_item and pam_open_session on PamHandle. Also define some nice constants.

### CODE REVIEW

**Comment by minrk** [comment]
Figured it out in #15, so if you add tests here, they should start passing if they work

**Comment by hardening** [comment]
Would be nice if the pip package could be updated after merge, so that a simple `pip install pamela`would install the last version.

File: pamela.py [hunk 1]
```diff
@@ -114,6 +144,11 @@ def get_envlist(self, encoding='utf-8'):
                 break
         return parsed
 
+    def open_session(self):
```
Comment (minrk) [inline pamela.py]: Presumably open_session should get a corresponding close_session?

Comment (hardening) [reply pamela.py]: Yeah right, my project that is using pamela should need that later.

**Comment by hardening** [comment]
Function `close_session` added.

**Comment by minrk** [comment]
Thanks! pamela-1.0 published to PyPI

**Comment by hardening** [comment]
nice !

**Pull request merged**